app/service/service-token-generator.js

```diff
@@ -5,7 +5,7 @@ const fetch = require('../util/fetch');
 
 const idamS2SUrl = config.get('idam.s2s_url');
 const serviceName = config.get('idam.service_name');
-const secret = config.get('idam.service_key');
+const secret = config.get('secrets.s2s.microservicekey-ccd-gw');
```

Comment:
Well, not a comment at the heart of the logic of the work, but miscroservicekey should be microservice-key. Or, do we really need to mention "microservice"? If this is something under s2s, the rest may omit microservice at all.

Reply:
its coming from the s2s vault, its an exact match on the name of the key in vault

Reply:
This is just the name of the key in the vault.  In this case the `s2s` vault has a key called `microservicekey-ccd-gw`.  So not much of an option and this follows the names of all the s2s microservice keys in that vault